src/main/fc/rc_controls.c

```diff
@@ -228,6 +233,14 @@ void processRcStickPositions()
     }
     doNotRepeat = true;
 
+    
+#if defined(USE_USB_CDC_HID) && defined(USE_VCP)
```

Comment:
The check for `defined(USE_VCP)` is unnecessary here: https://github.com/betaflight/betaflight/blob/d3ad63a50bbb48c39a1a4d4d6ceaff4a95c9fba4/src/main/target/common_post.h#L195-L198

Reply:
Thanks, I missed that one.